go/pkg/launchpad/ipfs.go

```diff
@@ -0,0 +1,94 @@
+package launchpad
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"golang.org/x/exp/slices"
+
+	remoteClient "github.com/ipfs/boxo/pinning/remote/client"
+	"github.com/ipfs/go-cid"
+)
+
+type Provider string
+
+const (
+	Pinata Provider = "pinata"
+)
+
+type PinningService struct {
+	url         string
+	bearerToken string
+	client      *remoteClient.Client
+}
+
+func NewPinningService(provider Provider, bearerToken string) (*PinningService, error) {
+	var url string
+
+	switch provider {
+	case Pinata:
+		url = "https://api.pinata.cloud/psa"

```

Comment:
i like to have const data like this on top of the file in a const so we can reuse-it and modify it easily from one place.I know it's used juste one time here, just personal preference, feel free to do what you feel is right

Reply:
https://github.com/TERITORI/teritori-dapp/pull/1432/commits/f903116d8d8cb518db3713e1a06efe68505bf345#diff-f73b583cce205cebcc2f818a43c3ac2eb9178d3519cd1e4936229975e67b494bR19